src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("String 학습 테스트")
+public class StringTest {
+
+    @Test
+    @DisplayName("1,2 가 들어왔을 때 1과 2로 분리되어 리턴")
+    public void splitOfCommaValue(){
+        String commaValue = "1,2";
+        String[] splitResult = commaValue.split(",");
+        Assertions.assertThat(splitResult).containsExactly("1", "2");
+    }
+
+    @Test
+    @DisplayName("1 이 들어왔을 때 1만을 포함하는 배열로 리턴")
+    public void splitOfOneLengthString() {
+        String oneLengthValue = "1";
+        String[] splitResult = oneLengthValue.split(",");
+        Assertions.assertThat(splitResult).contains("1");
+    }
+
+    @Test
+    @DisplayName("(1,2) 값이 주어졌을 때 괄호를 제거하여 리턴")
+    public void withoutParenthesesOfString() {
+        String withParentheses = "(1,2)";
+        int openParentheses  = 1;
+        int closeParentheses = 4;
+        String substring = withParentheses.substring(openParentheses, closeParentheses);
+        Assertions.assertThat(substring).isEqualTo("1,2");
```

Comment:
Assertions 의 정적 메서드는 테스트 코드에서 빈번하게 사용되는데요,
static import 를 통해 코드의 가독성을 조금 더 높여보면 어떨까요? 😄 
``` java
import static org.assertj.core.api.Assertions.*;
```
```suggestion
        assertThat(substring).isEqualTo("1,2");
```

Reply:
친절한 설명 감사합니다!
좋은 방법 같아요
다음 코드에서 적용 해 볼게요